src/main/resources/REST-README.md

```diff
@@ -0,0 +1,246 @@
+```
+  __  __ _                _____            __ _ _      
+ |  \/  (_)              |  __ \          / _(_) |     
+ | \  / |_  ___ _ __ ___ | |__) | __ ___ | |_ _| | ___ 
+ | |\/| | |/ __| '__/ _ \|  ___/ '__/ _ \|  _| | |/ _ \
+ | |  | | | (__| | | (_) | |   | | | (_) | | | | |  __/
+ |_|__|_|_|\___|_|  \___/|_|   |_|  \___/|_| |_|_|\___|
+  / ____| |           | |                              
+ | (___ | |_ __ _ _ __| |_ ___ _ __                    
+  \___ \| __/ _` | '__| __/ _ \ '__|                   
+  ____) | || (_| | |  | ||  __/ |                      
+ |_____/ \__\__,_|_|   \__\___|_|                      
+```
+
+# Project generator REST API
+
+This document summarizes examples of MicroProfile Starter web application REST API
+[microprofile/starter/1.0.0](https://app.swaggerhub.com/apis/microprofile/starter/1.0.0)
+
+Each MicroProfile version (mpVersion) offers a suite of specifications (specs) and is implemented
+by a set of application servers (supportedServers).
+
+In order to get a project zip generated one has to select at least the desired server (supportedServer).
+The latest mpVersion with all specs it offers is used by default then.
+
+# Get available MicroProfile versions
+
+```
+$ curl https://start.microprofile.io/api/1.0.0/mpVersion
+
+["MP22","MP21","MP20","MP14","MP13","MP12"]
+```
+
+# Get available supportedServers and specs
+
+Note the order of supportedServers values is pseudorandom each call.
+Note the JSON formatting was altered for this document.
+
+```
+$ curl https://start.microprofile.io/api/1.0.0/mpVersion/MP14
+{
+  "supportedServers": [
+    "PAYARA_MICRO",
+    "LIBERTY",
+    "TOMEE",
+    "KUMULUZEE"
```

Comment:
I suggest order the list alphabetically across the board.

Reply:
Hello @Emily-Jiang, thank you for the feedback. Let me comment below:

> Why THORNTAIL is not listed? Do we expect the latest version is used for all runtime?

[Thorntail is not listed as supporting MP14 in this application](https://github.com/Karm/microprofile-starter/blob/rest/src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/SupportedServer.java#L35)

> Will you have to update THORNTAIL image name when THORNTAIL_3 is released?

Whatever is in this [Enum](https://github.com/Karm/microprofile-starter/blob/rest/src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/SupportedServer.java#L31) appears in both Web and REST. It is not the point of this PR to change it at this point I would say. This is not REST specific - we use it for Web too and we have been doing so from the first Beta launch...

> I suggest order the list alphabetically across the board.

The order of servers is randomized on Web to avoid bias...I did the same for the REST API on purpose. I would like us to be consistent, so if we have REST in lexicographical order, we should do it for the Web too. I suggest a separate Issue and a PR discussion for that.

Reply:
ok